src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTask.java

```diff
@@ -325,4 +280,54 @@ private static String recordString(SinkRecord record) {
   public void close(Collection<TopicPartition> partitions) {
     offsetTracker.closePartitions(partitions);
   }
+
+  // Visible for testing
+  static class PartitionPauser {
+
+    // Kafka consumer poll timeout to set when partitions are paused, to avoid waiting for a long
+    // time (default poll timeout) to resume it.
+    private static final int PAUSE_POLL_TIMEOUT_MS = 100;
```

Comment:
`SinkTaskContext.timeout(...)` takes a `long`, so let's use a `long` here:
```suggestion
    private static final long PAUSE_POLL_TIMEOUT_MS = 100;
```